server/sitemap/generators/contentful-pages.js

```diff
@@ -0,0 +1,43 @@
+const fetchGraphQL = require('../../util/fetchGraphQL');
+const { info } = require('../../util/log');
+
+// Fetch all pages from contentful
+async function fetchContentfulPages() {
+	const result = await fetchGraphQL({
+		query: `{
+			contentful {
+				entries(contentType: "page", include: 0)
+			}
+		}`
+	}, 'data.contentful.entries');
+	const items = result?.items ?? [];
+
+	// Return all the pages with only the fields.key, fields.pageType, and fields.path properties
+	return items.map(({ fields }) => {
+		const { key, pageType, path } = fields;
+		return { key, pageType, path };
+	});
+}
+
+// This sitemap generator fetches all the pages in contentful and
+// adds a sitemap url for every route that is a landing page (/lp/*).
+module.exports = async function contentfulRouteGenerator() {
+	info('Sitemap: contentfulRouteGenerator: fetching contentful pages');
+	const pages = await fetchContentfulPages();
+
+	const landingPages = pages
+		// Only use 'landing' page types
+		.filter(({ pageType }) => pageType === 'landing')
+		// Use '/key' as the path if no path is defined
+		.map(({ key, path }) => path ?? `/${key}`)
+		// Only use pages with '/lp/*' paths
+		.filter(path => path.startsWith('/lp/'));
```

Comment:
@eddieferrer @emuvente Do you think we should also be including `/ub` routes? Seems like those will be tightly tied to social/ad efforts so it could be helpful to have them present...while they're in use at least.

Reply:
I wasn't sure about those because I didn't know what they were typically used for. I think that it's only marketing campaigns though, right? In which case I think it would make sense to include them in the sitemap, so that those campaigns can benefit from organic search traffic as well.

Reply:
Cool, doesn't have to be part of this PR. Could be something @eddieferrer can add in soon once those pages in use.